### PR TITLE
Homework - Webpack

### PR DESCRIPTION
**Pull request by vdumbrav**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/node_modules
+/yarn.lock
```
Comment (rapi) [inline .gitignore]: Don`t add lock file in gitignore

Comment (vdumbrav) [reply .gitignore]: ok

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```
Comment (rapi) [inline public/index.html]: dont put login on this file, it must be render by framework

File: webpack/webpack.config.prod.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env'],
+          },
+        },
+      },
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'eslint-loader',
```
Comment (rapi) [inline webpack/webpack.config.prod.js:34]: add eslint for vue files and style files

File: webpack/webpack.config.dev.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  },
+  devServer: {
+    hot: true,
+    open: true,
+  },
+  module: {
+    rules: [
+      {
+        test: /\.m?js$/,
+        exclude: /(node_modules|bower_components)/,
```
Comment (rapi) [inline webpack/webpack.config.dev.js]: where u use bower?

File: public/index.html (hunk 2 above)
Comment (vdumbrav) [reply public/index.html]: ok

File: webpack/webpack.config.dev.js (hunk 4 above)
Comment (vdumbrav) [reply webpack/webpack.config.dev.js]: sorry, copypast

File: webpack/webpack.config.prod.js (hunk 3 above)
Comment (vdumbrav) [reply webpack/webpack.config.prod.js:34]: ok

**Pull request closed without merging**